notebook/static/notebook/js/main.js

```diff
@@ -177,8 +171,28 @@ require([
       configurable: false
     });
 
-    utils.load_extensions_from_config(config_section);
-    utils.load_extensions_from_config(common_config);
+    // Now actually load nbextensions from config
+    Promise.all([
+        utils.load_extensions_from_config(config_section),
+        utils.load_extensions_from_config(common_config),
+    ])
+    .catch(function(error) {
+        console.error('Could not load nbextensions from user config files', error);
+    })
+    // BEGIN HARDCODED WIDGETS HACK
+    .then(function() {
+        if (!utils.is_loaded('widgets/extension')) {
```

Comment:
Now it is `jupyter-js-widgets/extension`. I opened the corresponding PR in master.